### ISSUE TITLE
Site Design

### ISSUE THREAD

**Issue by machuga** [coderie/coderie-site#1]
We need some sort of theming and logo for the site/group.

Current stack:
- Sass is our preprocessor of choice (.scss)
- Haml for templates
- Neat for grid system

Who wants the job?

**Comment by htmelvis** [comment]
working on the logo will work on theme as well

**Comment by matsilva** [comment]
gogo @htmelvis

**Comment by htmelvis** [comment]
## Quick List for Design
- [x] Start sketch file for base design wireframe
- [x] Color Scheme Chosen
- [x] Complete a logo design
- [ ] Solidify a basic styleguide 
- [ ] Icon set style for languages (indicators on blog posts, sections, etc.)

**Comment by htmelvis** [comment]
Can I have Access to the images that are located on our twitter along with any other images we may have? @coderie/owners

**Comment by Bshally** [comment]
Yeah those would also be helpful for issue #4 as requested in said issue.
